python/tvm/topi/x86/batch_matmul.py

```diff
@@ -162,10 +162,10 @@ def batch_matmul_blas_common(cfg, x, y, out_shape, lib):
     assert len(x.shape) == 3 and len(y.shape) == 3, "only support 3-dim batch_matmul"
     XB, M, XK = get_const_tuple(x.shape)
     YB, N, YK = get_const_tuple(y.shape)
-    assert XB == YB, "batch dimension doesn't match"
+    assert (XB == YB) or (YB == 1) or (XB == 1), "batch dimension doesn't match"
```

Comment:
1.From this statement it seems the case of `A(1, M, k) x B(8, N, K)` is supported. If so, please add a case to the test.
2. Please update the docstring. The shapes of x and y are no longer just `[batch, ...]`.

Reply:
1. Added this case. 2 Updated.